grunt_tasks/webpack.config.js

```diff
@@ -69,7 +70,14 @@ module.exports = {
             'window.jQuery': 'jquery'
         }),
         // Disable writing the output file if a build error occurs
-        new webpack.NoEmitOnErrorsPlugin()
+        new webpack.NoEmitOnErrorsPlugin(),
+        new GoogleFontsPlugin({
```

Comment:
I think it would be possible to import the google fonts css inside `global.styl` and remove the reference in the mako template.  As far as creating the file for each plugin, you could add the plugin for core only in [build.js](https://github.com/girder/girder/blob/master/grunt_tasks/build.js#L101).